public/app/plugins/datasource/azuremonitor/azure_monitor/url_builder.ts

```diff
@@ -50,8 +50,8 @@ export default class UrlBuilder {
     baseUrl: string,
     apiVersion: string,
     query: GetMetricNamespacesQuery,
-    globalRegion: boolean,
     templateSrv: TemplateSrv,
+    globalRegion?: boolean,
```

Comment:
nit: this function doesnt need `globalRegion` to be nullable. we shouldn't change this